apps/aecore/src/aec_txs_trees.erl

```diff
@@ -46,5 +46,5 @@ enter(K, V, T) ->
 
 enter_signed_tx(SignedTx, TxsTree) ->
     V = aec_tx_sign:serialize_to_binary(SignedTx),
-    K = aec_sha256:hash(V),
+    K = aec_hash:hash(tx, V),
```

Comment:
May I suggest signed_tx (or similar) rather than tx here? I would prefer keeping distinction of tx from the signed one for readability.